extensions/control-plane/provision/provision-gcs/src/main/java/org/eclipse/edc/connector/provision/gcp/GcsProvisionExtension.java

```diff
@@ -42,16 +37,16 @@ public String name() {
         return "GCP storage provisioner";
     }
 
+    @Inject
+    GcpManager gcpManager;
```

Comment:
nit: if the other injected components are `private`, please make this private as well, for consistency, and keep them together (so this should stay above the `name` method)

Reply:
Done